amethyst_renderer/src/system.rs

```diff
@@ -89,14 +89,7 @@ where
         );
     }
 
-    fn do_render(
-        &mut self,
-        (mut event_handler, mut window_messages, mut screen_dimensions, data): RenderData<P>,
-    ) {
-        #[cfg(feature = "profiler")]
-        profile_scope!("render_system");
-        use std::time::Duration;
-
+    fn window_management(&mut self, (mut window_messages, mut screen_dimensions): WindowData) {
```

Comment:
It might be worth noting that this PR changes the profiling in a way such that window management isn't included anymore. Maybe we should have an extra scope for it?

Reply:
Oh yeah that is important.  Thanks!

Reply:
Done.